saltstack/states/github-actions/runner.sls

```diff
@@ -28,6 +31,7 @@ runner-download:
 
 runner-configure:
   cmd.run:
+    # TODO: do we need to add a labels config here, or is the arch label automatic?
```

Comment:
I'm not sure how to test this out, @rawkode can you confirm whether or not the Arch label is applied automatically after the runner is added in this way?